packages/react/.storybook/main.js

```diff
@@ -85,9 +85,15 @@ module.exports = {
         },
         // Compiles Sass to CSS
         {
-          loader: 'fast-sass-loader',
+          loader: 'sass-loader',
           options: {
-            includePaths: [path.resolve(__dirname, '..', 'node_modules')],
+            implementation: require('sass'),
+            sassOptions: {
+              includePaths: [
+                path.resolve(__dirname, '..', 'node_modules'),
+                path.resolve(__dirname, '../../../node_modules'),
+              ],
+            },
```

Comment:
`fast-sass-loader` doesn't support `@use`, so we have to drop back to `sass-loader` and resolve both correct directory node_modules and root directory node_modules